scripts/system/libraries/entitySelectionTool.js

```diff
@@ -1451,14 +1479,15 @@ SelectionDisplay = (function() {
             isPointLight = properties.type == "Light" && !properties.isSpotlight;
         }
 
-        if (mode == "ROTATE_YAW" || mode == "ROTATE_PITCH" || mode == "ROTATE_ROLL" || mode == "TRANSLATE_XZ") {
+        if ( isActiveTool(yawHandle) || isActiveTool(pitchHandle) || 
+                isActiveTool(rollHandle) || isActiveTool(selectionBox) || isActiveTool(grabberCloner)) {
```

Comment:
I realize it's the way the old code was, but ... why is `rotationOverlaysVisible = true` for translate modes?  Seems like they maybe shouldn't be?